database/mongodb/README.md

```diff
@@ -75,62 +105,77 @@ That will start the MongoDB shell and from there you can create a user for examp
 
 ```
 
-In this example, we have created a user called `myUserAdmin` with a `userAdminAnyDatabase` role and with access to the `admin` database, this user has also as a password (`pwd`) the value `test`.
+In this example, we have created a user called `myUserAdmin` with a
+ `userAdminAnyDatabase` role and with access to the `admin` database, this user
+  has also as a password (`pwd`) the value `test`.
 
-After this you can exit the mongo shell and then `shell` into the container to access the database with the new created user.
+After this you can exit the mongo shell and then `shell` into the container to
+ access the database with the new created user.
 
 You can do so by:
 
-```
-$ sudo singularity shell mongodb.sif
+```sh
+singularity shell instance://mongodb
 ```
 
-The command prompt will open and then you can access the mongo database by running:
+The command prompt will open and then you can access the mongo database by
+ running:
 
-```
-$ mongod --repair
-$ mongod
-$ mongo --host $HOSTNAME:$PORT -u "myUserAdmin" -p "test"
+```sh
```

Comment:
Besides shell prompt. Is there a reason for removing the other commands?

Reply:
they are ran on the startscript

Reply:
No... it's a shell ... look at line 118

Reply:
```
$ mongod --repair
$ mongod
```
restart mongo, but those lines are run in the runscript, we only need to run the last line IF needed

I am prepping Carls demo, and I was able to boot mongo with out those lines